custom_components/zaptec/api.py

```diff
@@ -616,52 +641,44 @@ async def state(self):
             _LOGGER.debug("Access denied to charger %s state", self.id)
 
-        # Firmware version is called. SmartMainboardSoftwareApplicationVersion,
-        # stateid 908
-        # I couldn't find a way to see if it was up to date..
-        # maybe remove this later if it dont interest ppl.
-
-        if (installation_id := self["installation_id"]) in self.zaptec:
-            try:
-                firmware_info = await self.zaptec.request(
-                    f"chargerFirmware/installation/{installation_id}"
-                )
-                for fm in firmware_info:
-                    if fm["ChargerId"] == self.id:
-                        self.set_attributes(
-                            {
-                                "current_firmware_version": fm["CurrentVersion"],
-                                "available_firmware_version": fm["AvailableVersion"],
-                                "firmware_update_to_date": fm["IsUpToDate"],
-                            }
-                        )
-            except RequestError as err:
-                if err.error_code != 403:
-                    raise
-                _LOGGER.debug("Access denied to charger %s firmware info", self.id)
-
     #   API METHODS
     # =======================================================================
 
-    async def charger_info(self) -> TDict:
-        """Get the charger info."""
-        data = await self.zaptec.request(f"chargers/{self.id}")
-        return data
-
     async def command(self, command: str | int):
-        """Send a command to the charger."""
+        """Send a command to the charger.
 
-        if command == "authorize_charge":
-            return await self.authorize_charge()
+        Any command or command id can be used. Zaptec supports a number of
+        commands, which is found https://api.zaptec.com/help/index.html
+        under CommandId shema. The most used commands are:
 
-        self.is_command_valid(command, raise_value_error_if_invalid=True)
+        - deauthorize_and_stop: Deauthorize the charger and stop it
+        - restart_charger: Restart the charger
+        - resume_charging: Resume charging
+        - stop_charging_final: Stop charging and set final stop
+        - upgrade_firmware: Upgrade the firmware
+
+        Special commands which is special to this implementation:
+        - authorize_charge: Authorize the charger to charge
+        """
 
+        if command in ("authorize_charge", "AuthorizeCharge"):
+            return await self.authorize_charge()
+
+        # Look up the command and its command id
         if isinstance(command, int):
             # If int, look up the command name
             cmdid = command
-            command = ZCONST.commands.get(command)
+            command = ZCONST.commands.get(cmdid)
         else:
-            cmdid = ZCONST.commands.get(command)
+            # Support using the CommandName as a string
+            cmdid = ZCONST.commands.get(to_under(command))
```

Comment:
```suggestion
            command = to_under(command)
            cmdid = ZCONST.commands.get(command)
```
Otherwise we need to cover both PascalCase and snake_case in is_command_valid (or run to_under on the input to is_command_valid if that is preferable).

Reply:
Yes, I think all current uses are snake_case, and cleaning up the mix of Pascal and snake would definitely make it easier going forward.